finaps/libs/frontend/feat/home/src/lib/home/home.component.css

```diff
@@ -0,0 +1,7 @@
+:host {
+  --brand-primary: #420420;
```

Comment:
beautiful

Reply:
but also in scss variables a bit different ;)

Reply:
naturally